odoo/cli/command.py

```diff
@@ -39,43 +49,58 @@ def parser(self):
             )
         return self._parser
 
+    @classmethod
+    def is_valid_name(cls, name):
+        return re.match(COMMAND_NAME_RE, name)
+
 
 def load_internal_commands():
-    """Load `commands` from `odoo.cli`"""
+    """ Load ``commands`` from ``odoo.cli`` """
     for path in odoo.cli.__path__:
         for module in Path(path).iterdir():
             if module.suffix != '.py':
                 continue
             __import__(f'odoo.cli.{module.stem}')
 
 
-def load_addons_commands():
-    """Load `commands` from `odoo.addons.*.cli`"""
-    logging.disable(logging.CRITICAL)
+def load_addons_commands(command=None):
+    """
+    Search the addons path for modules with a ``cli/{command}.py`` file.
+    In case no command is provided, discover and load all the commands.
+    """
+    if command is None:
+        command = '*'
+    elif not Command.is_valid_name(command):
+        raise ValueError(command)
```

Comment:
before this, running a command not matching your regex would stop with simply `unknown command '…'`, now it’s producing a traceback

Reply:
Ok, I also added some call to action for the user.

![immagine](https://github.com/user-attachments/assets/5a1ec77b-524b-4131-adb0-d58299834f84)

Reply:
@mart-e ^^ is ok now?